tests/test-cases/Group1-Docker-Commands/1-06-Docker-Run.robot

```diff
@@ -133,3 +133,13 @@ Docker run verify name and id are not conflated
     ${rc}  ${container2}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -itd --name ${shortID1} busybox
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${container2}  Conflict
+
+Docker run verify anonymous volume contains files
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run jakedsouza/group-1-06-docker-verify-volume-files:1.0 ls /etc/example
+	Should Be Equal As Integers  ${rc}  0
+	Should Contain  ${output}  thisshouldexist
+	Should Contain  ${output}  testfile.txt
+
+	${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run jakedsouza/group-1-06-docker-verify-volume-files:1.0 cat /etc/example/testfile.txt
+    Should Be Equal As Integers  ${rc}  0
+	Should Contain  ${output}  TestFile
```

Comment:
IIRC you said that if a volume is explicitly specified with `-v name:/path` then this copy behaviour doesn't occur.
Please add a test for this case as well.